CONTRIBUTING.md

```diff
@@ -51,6 +51,14 @@ Below is a short excerpt of some points in there:
 
 See the nixpkgs manual for more details on [standard meta-attributes](https://nixos.org/nixpkgs/manual/#sec-standard-meta-attributes).
 
+## First Time contributors
+
+You will need to add yourself to [maintainers](https://nixos.org/manual/nixpkgs/unstable/#var-meta-maintainers)
+
+Your pr should have two commits:
+- `maintainers: add lizelive`
+- `starsector: 0.95.1a-RC6 -> 0.96a-RC7`
```

Comment:
Please make it explicit that this are examples or do template style wording.

Reply:
Here are the commits in an example [PR](https://github.com/NixOS/nixpkgs/pull/230958) where a user 'lizelive' updates 'starsector':